solr/schema.xml

```diff
@@ -86,7 +86,7 @@
    <field name="level" type="string" indexed="true" stored="true" multiValued="false"/>
 
    <!-- Accession properties required for facets -->
-   <field name="accession_date_year" type="int" indexed="true" stored="true" multiValued="false" />
+   <field name="accession_date_year" type="sort_string" indexed="true" stored="true" multiValued="false" />
```

Comment:
@sem42198 Won't this change cause a problem with the change you made in #1515? Perhaps leave this as an int and in indexer_common.rb, index it as 9999?